mscore/data/shortcuts.xml

```diff
@@ -491,6 +491,14 @@
     <seq>+</seq>
     </SC>
   <SC>
+    <key>pad-raise</key>
+    <seq>NumPad++</seq>
```

Comment:
OK, guess I'd have to try it out then. Would hate to loose the + for tie on the num-pad